gtsfm/utils/geometry_comparisons.py

```diff
@@ -248,3 +248,31 @@ def compute_points_distance_l2(wti1: Optional[Point3], wti2: Optional[Point3]) -
     if wti1 is None or wti2 is None:
         return None
     return np.linalg.norm(wti1 - wti2)
+
+
+def get_points_within_radius_of_cameras(wTi_list: List[Pose3], points_3d: np.ndarray, radius: float = 50) -> Optional[np.ndarray]:
+    """
+    Args:
```

Comment:
Doesnt the function need a one-line description before the args section?

Reply:
good point, I'll add it